dataland-frontend/tests/e2e/specs/low/pages/taxonomy/Paginator.ts

```diff
@@ -2,8 +2,8 @@ describe("Paginator test suite", () => {
   beforeEach(() => {
     cy.restoreLoginSession();
   });
-  it("Assure existence of paginator for Dax search", () => {
-    cy.visitAndCheckAppMount("/searchtaxonomy");
+  it("Assure existence of paginator for search to 'e'", () => {
```

Comment:
Ist das hier die richtige Lösung? Es gibt bereits den Test "Search for all companies containing a and verify that results are paginated, only first 100 are shown". Somit ist der Testfall bei der Suche schon mit abgedeckt und das wäre damit ein absolut doppelter Test

Reply:
Gut gesehen. Ich habe den Test aud "Datatables.ts" hierhin kopiert und Datatables.ts gelöscht.

Reply:
OK